client/my-sites/plugins/main.jsx

```diff
@@ -236,12 +244,23 @@ export default React.createClass( {
 	},
 
 	unselectOrSelectAll() {
+		const sites = this.props.sites.getSelectedOrAllWithPlugins().filter( site => this.state.bulkManagement.some( section => {
+			if ( section === 'jetpack' ) {
```

Comment:
can we extract this second embed function to a proper method with a name explaining what it does? I find it quite hard to understand without a cue :)